res/skins/LateNight/skin.xml

```diff
@@ -86,9 +88,35 @@
 
   <!-- MinimumSize should not be an exact monitor resolution. There needs
   to be space for the title bar or other chrome at full screen -->
-  <MinimumSize>1270,550</MinimumSize>
+  <MinimumSize>1344,668</MinimumSize>
```

Comment:
1302,668 
for Full HD Displays running Ubunty Unity.

Reply:
Than the EQ buttons are cutted with the Skin preferences visible though.
But the Mixxx is usable on my device now. 
If we consider the "preferences open" state as exceptional it should work.

Reply:
1366x768 is just HD (720+ lines)
actually I never really cared about squeezed skins with Pref pane open.
as long as the skin fits in its regular state where fine I think.